metpy/plots/tests/test_skewt.py

```diff
@@ -67,6 +67,20 @@ def test_skewt_with_grid_enabled():
         SkewT()
 
 
+@pytest.mark.mpl_image_compare(tolerance=0., remove_text=True)
```

Comment:
Will need a comparison image here as well.

Reply:
Is that something the test library takes care of, or do I need to upload an image somewhere?

Reply:
You generate it and add it to the PR - see the "image tests" section: https://github.com/Unidata/MetPy/blob/master/CONTRIBUTING.md

Reply:
... Or I could RTFM. (Sorry about that.)

Reply:
No worries- I'm not going to claim that the section is easy to find... Honestly wrote it mostly to remind myself 😄